src/gdpc/minecraft_tools.py

```diff
@@ -177,18 +2489,16 @@ def lecternData(bookData: Optional[str], page: int = 0) -> str:
 
 def bookData(
     text: str,
-    title: str         = "Chronicle",
-    author: str        = "Anonymous",
-    description: str   = "I wonder what's inside?",
-    desccolor: str     = "gold",
-    descIsItalic: bool = True
+    title: str                = "Chronicle",
+    author: str               = "Anonymous",
+    description: str          = "I wonder what's inside?",
+    descriptionColor: str     = "gold",
+    descriptionIsItalic: bool = True
```

Comment:
Good change, but changing parameter names is breaking in Python because of named parameter passing. So please revert for now.

```suggestion
    title: str         = "Chronicle",
    author: str        = "Anonymous",
    description: str   = "I wonder what's inside?",
    desccolor: str     = "gold",
    descIsItalic: bool = True
```